app/views/play/CampaignView.coffee

```diff
@@ -100,6 +100,7 @@ module.exports = class CampaignView extends RootView
     'shift+s': 'onShiftS'
 
   constructor: (options, @terrain) ->
+    debugger
```

Comment:
Debug cleanup 😉

Reply:
This merge commit has sneaked in. It is not supposed to be there. I am reverting the last commit. Trying to figure out how it happened.